frontend/src/Utils/ModelFeatureFlagLoader.test.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import ModelFeatureFlagLoader from './ModelFeatureFlagLoader';
+
+let setFeatureFlag = jest.fn();
+
+jest.mock('@openshift/dynamic-plugin-sdk', () => ({
+  useFeatureFlag: () => [undefined, setFeatureFlag],
+}));
+
+jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
+  useK8sModel: ({ group }: { group: string }) => [group === 'foo' ? {} : undefined],
+}));
+
+describe('ModelFeatureFlagLoader', () => {
+  test('should call feature flag once', () => {
+    setFeatureFlag = jest.fn();
+    render(
+      <ModelFeatureFlagLoader
+        flag="foo"
+        model={{
+          group: 'foo',
+          version: 'bar',
+          kind: 'baz',
+        }}
+      />,
+    );
+    expect(setFeatureFlag).toHaveBeenCalledTimes(1);
```

Comment:
@karelhala not blocking - but FWIW I think if we have the same exact test setup and props being passed into the function, we can make multiple assertions in one test. So:
```js
expect(setFeatureFlag).toHaveBeenCalledTimes(1);
expect(setFeatureFlag).toHaveBeenCalledWith(true);
```

Otherwise, for example, if we change `model` so `kind` is now `kindFoo` - we'd need to update both locations.